library/ssl_tls13_generic.c

```diff
@@ -1400,37 +1404,17 @@ int mbedtls_ssl_reset_transcript_for_hrr(mbedtls_ssl_context *ssl)
 
     hash_len += 4;
 
-#if defined(MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA)
-    if (ciphersuite_info->mac == MBEDTLS_MD_SHA256) {
-        MBEDTLS_SSL_DEBUG_BUF(4, "Truncated SHA-256 handshake transcript",
```

Comment:
As a result of this change we don't log `hash_transcript` anymore. Just checking if that is intended/if it's okay.

Reply:
Ah no, that wasn't intended, thanks for noticing! I think I had a brain bug where I thought this was just printing "Truncated SHA-xxx handshake transcript" which would have bee redundant with the message printed at when entering the function (and surely the value of xxx in SHA-xxx has been printed before), but it's also printing the contents for the hash transcript, which might be useful if we ever have bugs about it.

I'm restoring it.